ccmlib/common.py

```diff
@@ -328,6 +328,13 @@ def is_win():
     return sys.platform in ("cygwin", "win32")
 
 
+def is_modern_windows_install(version):
```

Comment:
So, I'm a little worried about type handling here. Some of the callsites we replaced were using strings, and others used ints. Could you add unit tests for this?